crates/rpc/rpc-layer/src/jwt_validator.rs

```diff
@@ -47,8 +47,13 @@ fn get_bearer(headers: &HeaderMap) -> Option<String> {
     let header = headers.get(header::AUTHORIZATION)?;
     let auth: &str = header.to_str().ok()?;
     let prefix = "Bearer ";
-    let index = auth.find(prefix)?;
-    let token: &str = &auth[index + prefix.len()..];
+
+    // Ensure the header starts with "Bearer " to prevent bypass attacks
```

Comment:
can we remove this comment, this doesnt have any relevance without the context of this pr